consensus/spos/commonSubround/subroundBlock.go

```diff
@@ -87,6 +87,10 @@ func (sr *SubroundBlock) doBlockJob() bool {
 		return false
 	}
 
+	sr.BlockSizeThrottler().Add(
+		uint64(sr.RoundIndex),
+		core.Max(sr.Header.ItemsInBody(), sr.Header.ItemsInHeader()))
```

Comment:
this can be called at createBlock / processBlock time

Reply:
Done